### ISSUE TITLE
Correctly fall back on missing shapes.txt

### ISSUE THREAD

**Issue by irees** [transitland/transitland-datastore#339]
Test Caltrain with new gtfs gem improvements.

**Comment by irees** [comment]
@doublestranded in branch 'update-gtfs-gem', which includes the fixes in the gtfs wrapper, I'm getting an RSP import error. If you could take a look when you get a chance.

```
2016-01-29T00:22:07.621Z 41135 TID-ova3jvsp4 INFO: Load GTFS
2016-01-29T00:22:07.633Z 41135 TID-ova3jvsp4 INFO: Calculating distances
D, [2016-01-28T16:22:07.634446 #41135] DEBUG -- :   RouteStopPattern Load (0.2ms)  SELECT  "current_route_stop_patterns".* FROM "current_route_stop_patterns" WHERE "current_route_stop_patterns"."onestop_id" = $1 LIMIT 1 /*application:TransitlandDatastore,line:/app/models/concerns/has_a_onestop_id.rb:10:in `find_by_onestop_id!'*/  [["onestop_id", "r-9q9k6-shuttle-aa7144-9a837d"]]
2016-01-29T00:22:07.634Z 41135 TID-ova3jvsp4 ERROR:
Couldn't find RouteStopPattern
["/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activerecord-4.2.5.1/lib/active_record/core.rb:196:in `find_by!'", "/Users/irees/mapzen/transitland-datastore/app/models/concerns/has_a_onestop_id.rb:10:in `find_by_onestop_id!'", "app/services/gtfs_graph.rb:77:in `block in ssp_perform_async'", "app/services/gtfs_graph.rb:76:in `each'", "app/services/gtfs_graph.rb:76:in `ssp_perform_async'", "/Users/irees/mapzen/transitland-datastore/app/workers/feed_eater_schedule_worker.rb:15:in `perform'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/testing.rb:289:in `execute_job'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/testing.rb:284:in `block in process_job'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/middleware/chain.rb:127:in `block in invoke'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/middleware/chain.rb:132:in `call'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/middleware/chain.rb:132:in `invoke'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/testing.rb:283:in `process_job'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sidekiq-4.0.2/lib/sidekiq/testing.rb:267:in `drain'", "app/services/gtfs_graph.rb:540:in `<top (required)>'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/runner.rb:60:in `load'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/runner.rb:60:in `<top (required)>'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:123:in `require'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:123:in `require_command!'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:90:in `runner'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'", "/Users/irees/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.1/lib/rails/commands.rb:17:in `<top (required)>'", "bin/rails:4:in `require'", "bin/rails:4:in `<main>'"]
```

**Comment by irees** [comment]
Resolved by #356
